pkg/object/objects.go

```diff
@@ -279,6 +291,32 @@ func (os K8sObjects) JSONManifest() (string, error) {
 	return b.String(), nil
 }
 
+// YAMLManifest returns a YAML representation of K8sObjects os.
+func (os K8sObjects) YAMLManifest() (string, error) {
+	var b bytes.Buffer
+
+	for i, item := range os {
+		if i != 0 {
+			if _, err := b.WriteString("\n\n"); err != nil {
```

Comment:
why this is necessary since we already write YAMLSeparator at the end

Reply:
This is because the object YAMLs don't have separators. We only have those in the charts but they disappear after being parsed in to objects.